extensions/amp-delight-player/0.1/amp-delight-player.js

```diff
@@ -240,17 +246,28 @@ class AmpDelightPlayer extends AMP.BaseElement {
    * @private
    */
   handleDelightMessage_(event) {
-    if (event.source !== this.iframe_.contentWindow) {
+    if (!originMatches(event, this.iframe_, /.*/)) {
       return;
     }
 
     const data = objOrParseJson(getData(event));
-    if (data === undefined || data['type'] === undefined) {
+    if (data === null || data === undefined || data['type'] === undefined) {
       return; // We only process valid JSON.
     }
 
     const {element} = this;
 
+    redispatch(element, data['type'], {
```

Comment:
`redispatch` returns a boolean to short-circuit the message method, can you return?

```js
const redispatched = redispatch(...);
if (redispatch) {
  return;
}
```